### PR TITLE
Build wheel and add a script to extract to local system

### PR DESCRIPTION
**Pull request by BetsyMcPhail**
This commit will provide develop a one-step operation to create a python wheel.

The wheel itself is still a WIP, but this should be a starting place for debugging.

### CODE REVIEW

**Comment by BetsyMcPhail** [comment]
The `auditwheel repair` command is failing and so, even though the wheel is created, it is not transferred to the local machine.

```
#20 1640.4 INFO:auditwheel.main_repair:Repairing pydrake-0.32.0-py3-none-any.whl
#20 1656.7 usage: auditwheel [-h] [-V] [-v] command ...
#20 1656.7 auditwheel: error: cannot repair "dist/pydrake-0.32.0-py3-none-any.whl" to "manylinux_2_5_x86_64" ABI because of the presence of too-recent versioned symbols. You'll need to compile the wheel on an older toolchain.
#20 ERROR: executor failed running [/bin/sh -c /bin/bash /image/build-wheel.sh]: exit code: 2
```

**Comment by BetsyMcPhail** [comment]
After a pip install pydrake, the error I'm seeing is:

```
root@26f7b0479313:~# python3 -c 'import pydrake; print(pydrake.__file__)'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.6/dist-packages/pydrake/__init__.py", line 41, in <module>
    from . import common
  File "/usr/local/lib/python3.6/dist-packages/pydrake/common/__init__.py", line 7, in <module>
    from ._module_py import *
ImportError: libdrake.so: cannot open shared object file: No such file or directory

```

**Comment by BetsyMcPhail** [comment]
```
root@26f7b0479313:/build-wheel/dist# auditwheel show pydrake-0.32.0-py3-none-any.whl

pydrake-0.32.0-py3-none-any.whl is consistent with the following
platform tag: "linux_x86_64".

The wheel references external versioned symbols in these
system-provided shared libraries: libstdc++.so.6 with versions
{'GLIBCXX_3.4.11', 'CXXABI_1.3.3', 'CXXABI_1.3.2', 'CXXABI_1.3.11',
'GLIBCXX_3.4', 'GLIBCXX_3.4.19', 'GLIBCXX_3.4.21', 'CXXABI_1.3.5',
'GLIBCXX_3.4.18', 'GLIBCXX_3.4.9', 'GLIBCXX_3.4.22', 'GLIBCXX_3.4.20',
'CXXABI_1.3.8', 'GLIBCXX_3.4.15', 'GLIBCXX_3.4.14', 'CXXABI_1.3',
'CXXABI_1.3.9'}, libc.so.6 with versions {'GLIBC_2.15', 'GLIBC_2.3.4',
'GLIBC_2.3', 'GLIBC_2.2.5', 'GLIBC_2.6', 'GLIBC_2.17', 'GLIBC_2.8',
'GLIBC_2.3.2', 'GLIBC_2.11', 'GLIBC_2.14', 'GLIBC_2.4', 'GLIBC_2.7'},
libpthread.so.0 with versions {'GLIBC_2.3.2', 'GLIBC_2.2.5',
'GLIBC_2.3.3'}, libgcc_s.so.1 with versions {'GCC_3.0'}, libm.so.6
with versions {'GLIBC_2.27', 'GLIBC_2.2.5'}, libdl.so.2 with versions
{'GLIBC_2.2.5', 'GLIBC_2.3.3'}, libQt5PrintSupport.so.5 with versions
{'Qt_5'}, libQt5Gui.so.5 with versions {'Qt_5'}, libQt5Widgets.so.5
with versions {'Qt_5'}, libQt5Core.so.5 with versions {'Qt_5',
'Qt_5_PRIVATE_API', 'Qt_5.9'}, libgfortran.so.4 with versions
{'GFORTRAN_7'}, 
.so.62 with versions {'LIBJPEG_6.2'},
liblzma.so.5 with versions {'XZ_5.0'}, librt.so.1 with versions
{'GLIBC_2.2.5'}, libz.so.1 with versions {'ZLIB_1.2.9',
'ZLIB_1.2.3.4'}

This constrains the platform tag to "manylinux_2_27_x86_64". In order
to achieve a more compatible tag, you would need to recompile a new
wheel from source on a system with earlier versions of these
libraries, such as a recent manylinux image.

```

File: image/setup.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    def is_pure(self):
+        return False
+    def has_ext_modules(self):
+        return True
+
+
+setup(name='pydrake',
+      version=DRAKE_VERSION,
+      description='Model-based design and verification for robotics',
+      long_description ='''Drake ("dragon" in Middle English) is a toolbox maintained by the Robot
```
Comment (mwoehlke-kitware) [inline image/setup.py:49]: Why does this line, and *only* this line, have a space before the `=`?

Has flake8 or similar been run on this?

File: image/setup.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Locomotion Group at the MIT Computer Science and Artificial Intelligence
+Lab (CSAIL). It is a collection of tools for analyzing the dynamics of
+robots and building control systems for them in C++ and Python, with a
+heavy emphasis on optimization-based design/analysis.''',
+      url='https://drake.mit.edu',
+      author='Drake Development Team',
+      author_email='drake-users@mit.edu',
+      classifiers[
```
Comment (mwoehlke-kitware) [inline image/setup.py:57]: No `=` is needed here?

**Comment by mwoehlke-kitware** [comment]
@BetsyMcPhail, is this built in Docker? It seems not; there should not be any Qt5 libraries AFAIK/CT in the Docker build environment.

**Comment by BetsyMcPhail** [comment]
> @BetsyMcPhail, is this built in Docker? It seems not; there should not be any Qt5 libraries AFAIK/CT in the Docker build environment.

Yes, it seems weird to me too. Maybe I polluted my docker somehow? I'm going to try again

File: image/setup.py (hunk 1 above)
Comment (BetsyMcPhail) [reply image/setup.py]: It has now :) Should be better

File: Dockerfile [hunk 3]
```diff
@@ -36,5 +36,6 @@ ENV REPO=https://api.github.com/repos/robotlocomotion/drake
 ADD image/build-wheel.sh /image/
 ADD image/pip-drake.patch /image/
 ADD ${REPO}/git/refs/heads/master /tmp/drake.sha
+ADD image/setup.py /image/
```
Comment (mwoehlke-kitware) [inline Dockerfile:39]:
```suggestion
ADD image/setup.py /image/
ADD ${REPO}/git/refs/heads/master /tmp/drake.sha
```

**Comment by BetsyMcPhail** [comment]
Superseded by #4

**Pull request closed without merging**